src/vocgui/serializers.py

```diff
@@ -72,10 +86,13 @@ class Meta:
         Define model and the corresponding fields
         """
 
+        list_serializer_class = DocumentImageListSerializer
         model = DocumentImage
         fields = ("id", "image")
 
 
+
```

Comment:
Too many new lines here? 2 new lines should suffice. Everything else looks good.